README.md

```diff
@@ -628,6 +629,15 @@ export const Dialog = styled.dialog`
 `
 ```
 
+### Fonts
+First a few do's and don'ts:  
+- **DON'T** use fonts downloaded from a non-reliable source
+- **DON'T** use fonts converted by tools like Font Squirrel
+- **DO** host font files on the same domain to improve initial page load time.
```

Comment:
```suggestion
- **DO** host font files on the same domain to improve initial page load time – when in control of the font files
```

As there are cases where we rely on hosted licensed fonts, we cannot adhere to this rule.